commands/go_to_definition.py

```diff
@@ -0,0 +1,23 @@
+import sublime_plugin
+
+from ..libs.viewhelpers import *
+from ..libs.reference import *
+
+
+class TypescriptGoToDefinitionCommand(sublime_plugin.TextCommand):
+    """Go to definition command"""
+    def is_enabled(self):
+        return is_typescript(self.view)
+
+    def run(self, text):
+        check_update_view(self.view)
+        definitionResp = cli.service.definition(self.view.file_name(), get_location_from_view(self.view))
+        if definitionResp["success"]:
+            codeSpan = definitionResp["body"][0] if len(definitionResp["body"]) > 0 else None
```

Comment:
Also, call it `definition_location` or `definitionLoc` instead of `codeSpan`. Just keep in mind that we're trying to move to snake_case_naming  as far as I understand.